bindings/http/http.go

```diff
@@ -234,6 +245,16 @@ func (h *HTTPSource) Invoke(ctx context.Context, req *bindings.InvokeRequest) (*
 		request.Header.Set("Accept", "application/json; charset=utf-8")
 	}
 
+	// HTTP binding needs to inject traceparent header for proper tracing stack.
+	if traceHeaders {
+		if tp, ok := req.Metadata[traceparentHeader]; ok {
```

Comment:
Rather than checking if it existsC, here and in the next block please check if the value is not "". Otherwise this could set the header to an empty value, but present.